astropy/_erfa/interface.py

```diff
@@ -0,0 +1,177 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+Interfaces to the ERFA library, in particular for leap seconds.
+"""
+from datetime import datetime, timedelta
```

Comment:
It would be nice to use `Time` but that leads to round-trip issues when adding auto-updates upon loading of `_erfa`.

Reply:
None of this is actually *instantiated* at import time, right?  So can't we just use `time` imports *inside* the functions?  I think that's a more consistent approach.  

(If I missed an import-time requirement, then you can ignore this, though. It's not *that* bad since it's easy to transition between `Time` and `datetime` anyway)

Reply:
I wish!! (and tried). The problem is that `Time` initiation needs `erfa` installed (even the plainest formats like 'iso' rely on it). More specifically, `Time.now()` needs to have the leap seconds up to date (in case 'now' is on a leap-second day), yet here I need to have `Time.now()` accessible to tell whether to update the leap-second table.